src/main/java/thePackmaster/cards/pinnaclepack/PocketTournamentUncommonSkill.java

```diff
@@ -0,0 +1,29 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.pinnaclepack.actions.MinigameAction;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class PocketTournamentUncommonSkill extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("PocketTournamentUncommonSkill");
+    private static final int MAGIC = 0;
+    private static final int UPGRADE_MAGIC = 2;
+
+    public PocketTournamentUncommonSkill() {
```

Comment:
*Pocket Tournament*
Instead of the using a keyword, the effect of this card should be written out on the card.

This means that the cards text will read: "Randomly, 2X times: NL Draw 1 card next turn. NL Gain [E] next turn. NL Gain 3 Block next turn. NL Gain 5 Vigor. NL Exhaust." This is at the limit of 6 lines per card, and it's not great that even after these suggestions there will be multiple long/complex cards like this still around, but it will help.